freeipa.spec.in

```diff
@@ -534,14 +565,15 @@ Integrated DNS server is BIND 9. OpenDNSSEC provides key management.
 
 %package server-trust-ad
 Summary: Virtual package to install packages required for Active Directory trusts
-Group: System Environment/Base
 Requires: %{name}-server = %{version}-%{release}
 Requires: %{name}-common = %{version}-%{release}
 
 Requires: samba >= %{samba_version}
 Requires: samba-winbind
 Requires: libsss_idmap
-
+%if 0%{?rhel} < 9
```

Comment:
Why is this only needed for RHEL < 9?

Reply:
Because RHEL9 does not have and will not have ipa-idoverride-memberof-plugin.

Reply:
Is an upgarde from CentOS 8.2 to CentOS 9 supported as an example?

Reply:
My current understanding is that for major OS upgrade will be supported via replication for IPA servers. We will most likely have a change in internal idm:DL1 module dependencies that wouldn't be handled by DNF upgrade code. Either subsuming pki-core/pki-deps modular dependencies or switching their module streams -- e.g. pki-core:10.6 to pki-core:11 or so. This is currently not supported by DNF and we are still not clear when they are going to support this internal module dependency switch.

Reply:
I removed '< 9' and left only rhel check.